homeassistant/components/watergate/coordinator.py

```diff
@@ -14,7 +14,22 @@
 _LOGGER = logging.getLogger(__name__)
 
 
-class WatergateDataCoordinator(DataUpdateCoordinator[DeviceState]):
+class WatergateAgregatedRequests:
+    """Class to hold aggregated requests."""
+
+    def __init__(
+        self,
+        state: DeviceState,
+        telemetry: TelemetryData,
+        networking: NetworkingData,
+    ) -> None:
+        """Initialize aggregated requests."""
+        self.state = state
+        self.telemetry = telemetry
+        self.networking = networking
```

Comment:
```suggestion
@dataclass
class WatergateAgregatedRequests:
    """Class to hold aggregated requests."""
    
        state: DeviceState,
        telemetry: TelemetryData,
        networking: NetworkingData,
```
will give you the same